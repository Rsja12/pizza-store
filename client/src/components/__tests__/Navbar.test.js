import React from 'react';
import { shallow } from 'enzyme';

import { Navbar } from '../Navbar';

it('renders with empty cart', () => {
    const mockCart = [];

    expect(shallow(<Navbar cart={mockCart} />)).toMatchSnapshot();
});

it('renders with at least one item in cart', () => {
    const mockCart = [{}];

    expect(shallow(<Navbar cart={mockCart} />)).toMatchSnapshot();
});
