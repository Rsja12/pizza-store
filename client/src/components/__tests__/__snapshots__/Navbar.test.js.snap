// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders with at least one item in cart 1`] = `ShallowWrapper {}`;

exports[`renders with empty cart 1`] = `ShallowWrapper {}`;
